<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qync::PreferencesDialogue</class>
 <widget class="QDialog" name="Qync::PreferencesDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="rsyncPathLayout">
     <item>
      <widget class="QLabel" name="rsyncPathLabel">
       <property name="text">
        <string>rsync path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rsyncPath">
       <property name="text">
        <string>/usr/bin/rsync</string>
       </property>
       <property name="placeholderText">
        <string>Path to rsync executable...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="chooseRsyncPath">
       <property name="toolTip">
        <string>Open a file dialogue to choose the rsync executable.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="simpleUi">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The simple user interface is suitable for simple backup operations. The full user interface provides access to more advanced options.&lt;/p&gt;&lt;p&gt;You can temporarily switch between the UIs using &lt;span style=&quot; font-weight:600;&quot;&gt;Ctrl-U, S&lt;/span&gt; for the simple UI or &lt;span style=&quot; font-weight:600;&quot;&gt;Ctrl-U, F&lt;/span&gt; for the fullUI. Switching this way does not alter your preferences.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Simple user interface</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="toolbarGroup">
     <property name="title">
      <string>Toolbars</string>
     </property>
     <layout class="QGridLayout" name="toolbarsLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="presetsToolbar">
        <property name="text">
         <string>Presets</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="synchroniseToolbar">
        <property name="text">
         <string>Synchronise</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="toolbarStyleLayout">
        <item>
         <widget class="QLabel" name="toolbarStyleLabel">
          <property name="text">
           <string>Style</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="toolbarStyle">
          <item>
           <property name="text">
            <string>Style default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Icons only</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Text under icons</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Text beside icons</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Text only</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="mainSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="controls">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>savePreferences()</slot>
  <slot>chooseRsyncPath()</slot>
 </slots>
</ui>
